osbuild/util/chroot.py

```diff
@@ -13,7 +13,7 @@ class Chroot:
     """
 
     def __init__(self, root: str):
-        self.root = root
+        self.root = str(root)
```

Comment:
The usual way to do this is `os.fspath()` as this will take __fspath__() into consideration

Reply:
This should not be needed since `chroot` is supposed to be `str`. I'd say that the caller needs to make sure to pass `str` as an argument.

Reply:
Good catch!

Reply:
This looks to be resolved now.